src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveMinimalUnaryOpTestTemplate.template

```diff
@@ -294,6 +297,17 @@ namespace JIT.HardwareIntrinsics.Arm
             ConditionalSelectScenario_FalseValue({RetVectorType}<{RetBaseType}>.AllBitsSet, _fld1, {RetVectorType}<{RetBaseType}>.Zero);
         }
 
+        public void RunScenario_AbsorbConditionalSelectOp()
+        {
+            TestLibrary.TestFramework.BeginScenario("RunScenario_AbsorbConditionalSelectOp - zero in falseValue");
+            var mask = Vector.As<{RetBaseType}, {Op1BaseType}>(_mask);
+            var zeroVec = {Op1VectorType}<{Op1BaseType}>.Zero;
+            var result = {Isa}.{Method}(Sve.ConditionalSelect(mask, _fld1, zeroVec));
```

Comment:
we do not support predicate version of `sel` yet, right? otherwise something like `Sve.ConditionalSelect(mask, mask1, mask2)` could be one more test scenario.

Reply:
True. We don't support the predicate version of sel yet.